package.json

```diff
@@ -110,24 +110,35 @@
     "@keystonehq/bc-ur-registry-eth": "^0.12.1",
     "@keystonehq/metamask-airgapped-keyring": "^0.6.1",
     "@material-ui/core": "^4.11.0",
+    "@metamask/address-book-controller": "1.0.0-f2fa748",
```

Comment:
The versions we are using here look a bit funky, but they're autogenerated based on [this PR](https://github.com/MetaMask/controllers/pull/960) and are temporary until we are able to QA the new packages in both extension and mobile. Once we've done that I'll replace them with "1.0.0".